include/constants/battle_partner.h

```diff
@@ -0,0 +1,11 @@
+
+#ifndef GUARD_CONSTANTS_BATTLE_PARTNERS_H
+#define GUARD_CONSTANTS_BATTLE_PARTNERS_H
+
+#define PARTNER_NONE                0
+#define PARTNER_STEVEN              1
+#define PARTNER_MAY                 2
+
+#define PARTNERS_COUNT              3
```

Comment:
```suggestion

#define PARTNERS_COUNT              1
```
`PARTNERS_COUNT` is not used at all

Reply:
I used it in `GetTrainerNameFromId` but found a different way to handle it. Should it not just be removed?

Reply:
Applied the change as you requested for now.